front/components/actions/tables_query/TablesQueryActionDetails.tsx

```diff
@@ -29,18 +29,8 @@ export function TablesQueryActionDetails({
       visual={TableIcon}
     >
       <div className="flex flex-col gap-1 gap-4 pl-6 pt-4">
-        <div className="flex flex-col gap-1">
-          <span className="text-sm font-bold text-slate-900">Reasoning</span>
-          <div className="text-sm font-normal text-slate-500">
-            <QueryThinking action={action} />
-          </div>
-        </div>
-        <div className="flex flex-col gap-1">
-          <span className="text-sm font-bold text-slate-900">Query</span>
-          <div className="text-sm font-normal text-slate-500">
-            <TablesQuery action={action} />
-          </div>
-        </div>
+        <QueryThinking action={action} />
```

Comment:
I don't quite see the hiding logic but code chance lgtm

Reply:
We return null if the thing is missing. So the header and paddings are no longer rendered

Reply:
Ah I see makes sense :+1: